apps/minifront/src/components/swap/auction-list/helpers.ts

```diff
@@ -20,9 +20,9 @@ export const filterWithLimit = <T>(
   return result;
 };
 
-export const byStartHeightAscending = (a: AuctionInfo, b: AuctionInfo) => {
+export const byStartHeightDescending = (a: AuctionInfo, b: AuctionInfo) => {
```

Comment:
Hmm. I think you may be right. Perhaps we should keep as is then. In the case where there has been a bunch not withdrawn, this makes more sense. But think users will be motivated to withdraw their stuff.

Reply:
Users are probably motivated to withdraw auctions, otherwise, their funds will be locked in auctions and cannot be used for other auctions or other activities

I think it would be more convenient for the "active" tab to add newly created auctions to the end of the list (i.e. sorting in ascending order). 
But for the "all" tab, it is obviously more convenient to display new auctions above (i.e. sorting in descending order)
If you choose one sorting for all tabs, then sorting by descending is the best option for me.

Reply:
Let's leave as descending for now and can revisit later